rust/src/encodings/plain.rs

```diff
@@ -242,6 +250,16 @@ impl<'a> PlainDecoder<'a> {
             })?;
         Ok(Arc::new(FixedSizeBinaryArray::try_new(values, stride)?) as ArrayRef)
     }
+
+    async fn take_boolean(&self, indices: &UInt32Array) -> Result<ArrayRef> {
+        // TODO: optimize boolean access
```

Comment:
make a ticket for this?

Reply:
Filed https://github.com/eto-ai/lance/issues/629